index.test.js

```diff
@@ -0,0 +1,74 @@
+const http = require('http');
+const supertest = require('supertest');
+const handler = require('./src/handler');
+const parseEnv = require('./src/parse-env');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+
+const getExistingPosts = async () => {
+  const config = parseEnv(process.env);
+  const server = http.createServer(handler(config));
+
+  return (await supertest(server).get('/v0/Posts')).body;
+};
+
+describeIfEnv('integration', () => {
+  this.config = parseEnv(process.env);
+
+  describe('allowedMethods "*"', () => {
+
+    beforeEach(() => {
+      this.config.allowedMethods = '*';
```

Comment:
Use `let` instead of `this`. `this` can create hard-to-debug dependencies between tests. For instance:

```js
describe('micro-airtable-api', () => {
  describe('thing one', () => {
    this.server = http.createServer();
  });

  describe('thing two', () => {
    // if we forget to redefine `this.server` do we have access to it here?
  });
});
```

Instead, using `let` or `const` ensures we're much more explicit about what's being shared:

```js
describe('micro-airtable-api', () => {
  describe('thing one', () => {
    const config = { ... };
    let server;

    beforeAll(() => {
      server = http.createServer();
    });
  });

  describe('thing two', () => {
    // we can see `server` and `config` aren't defined here
  });
});
```